apps/extension/src/routes/page/index.tsx

```diff
@@ -11,18 +11,10 @@ import { getDefaultFrontend } from '../../state/default-frontend';
 // Will redirect to onboarding if necessary.
 export const pageIndexLoader = async () => {
   const wallets = await localExtStorage.get('wallets');
-  const grpcEndpoint = await localExtStorage.get('grpcEndpoint');
-  const frontendUrl = await localExtStorage.get('frontendUrl');
 
   if (!wallets.length) {
     return redirect(PagePath.WELCOME);
   }
-  if (!grpcEndpoint) {
-    return redirect(PagePath.SET_GRPC_ENDPOINT);
-  }
-  if (!frontendUrl) {
-    return redirect(PagePath.SET_DEFAULT_FRONTEND);
-  }
```

Comment:
Checks no longer necessary as we now set all of these at once and not separate steps